client-misk/src/test/kotlin/app/cash/backfila/client/misk/ClientMiskTestingModule.kt

```diff
@@ -48,10 +47,14 @@ internal class ClientMiskTestingModule(
     install(LogCollectorModule())
     install(MiskTestingServiceModule())
     install(EmbeddedBackfilaModule())
-    install(BackfilaModule(
-        BackfilaClientConfig(
-            url = "test.url", slack_channel = "#test"),
-        backfillClasses
-    ))
+    install(
+        BackfilaModule(
+            BackfilaClientConfig(
+                url = "test.url", slack_channel = "#test"
+            )
+        )
+    )
+
+    binder().bindBackfill<TestBackfill>()
```

Comment:
eg

Reply:
Yeah this reified version looks good. 

We probably want to log what gets registered. I think this already happens on the backfila side though.